MVP/Resources/Translations.es.resx

```diff
@@ -0,0 +1,561 @@
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="contributions_title" xml:space="preserve">
+    <value>Actividades de la comunidad</value>
+  </data>
+  <data name="search_placeholder" xml:space="preserve">
+    <value>Buscando...</value>
+  </data>
+  <data name="profile_title" xml:space="preserve">
+    <value>Configuraciones</value>
+  </data>
+  <data name="setting_contributionfromurl_title" xml:space="preserve">
+    <value>Crear contribución desde URL</value>
+  </data>
+  <data name="setting_contributionfromurl_description" xml:space="preserve">
+    <value>Si tienes una URL válida en tu portapapeles mientras agregas una contribución, se te preguntará si los metadatos de esa URL deben usarse para completar tu contribución.</value>
+  </data>
+  <data name="setting_apptheme_title" xml:space="preserve">
+    <value>Tema de la aplicación</value>
+  </data>
+  <data name="setting_apptheme_description" xml:space="preserve">
+    <value>Esta aplicación se puede utilizar tanto en modo claro como oscuro. De forma predeterminada, elegirá lo que esté configurado en tu dispositivo. Si alguno de los temas te gusta, también puedes forzar un tema específico aquí.</value>
```

Comment:
I would suggest using the term "establecer" instead of "forzar" in the line "también puedes forzar un tema específico aquí